open_feature/open_feature_client.py

```diff
@@ -359,3 +362,24 @@ def _create_provider_evaluation(
             raise TypeMismatchError()
 
         return value
+
+    def __extract_evaluation_options(
+        self, flag_evaluation_options: typing.Any
```

Comment:
We have a type for `FlagEvaluationOptions` now.

Reply:
Now that this type should be passed directly I'm removing the method to extract it from a dictionary.

Reply:
Either I'm misunderstanding your comment, or there's a commit you haven't pushed. :sweat_smile:

Reply:
Definitely the latter- and it appears I had done that work on my desktop instead of my thinkpad so it'll be another day before I push it up.

Reply:
Definitely the latter. I re-implemented it just now since the commit doesn't appear to have been on my thinkpad :smile: